.travis.yml

```diff
@@ -2,19 +2,19 @@ language:
     csharp
 
 install:
-    - sudo apt-get install nunit-console
+#    - sudo apt-get install nunit-console
```

Comment:
Why is this commented? This will cause the Travis CI check to fail.

Reply:
Made changes